dll/win32/shell32/wine/shellpath.c

```diff
@@ -103,6 +103,13 @@ DoGetProductType(PNT_PRODUCT_TYPE ProductType)
     return TRUE;
 }
 
+BOOL APIENTRY IsRemovableDrive(DWORD iDrive)
+{
+    WCHAR szRoot[] = L"C:\\";
+    szRoot[0] = (WCHAR)(L'A' + iDrive);
+    return GetDriveTypeW(szRoot) == DRIVE_REMOVABLE;
```

Comment:
Should `DRIVE_CDROM` also be checked here?

Reply:
Windows doesn't check `DRIVE_CDROM` here.